docs/API.md

```diff
@@ -542,6 +542,8 @@ along with any other options: `['node_modules', 'bower_components']`
 
 A map from regular expressions to module names that allow to stub out resources, like images or styles with a single module.
 
+When imported, these mapped modules come automatically unmocked.
```

Comment:
Can we instead say "Modules that are mapped to an alias are unmocked by default, regardless of whether automocking is enabled or not." ?